pydata/urls.py

```diff
@@ -0,0 +1,14 @@
+from django.conf.urls import url
+
+from . import views
+
+
+urlpatterns = [
+    url(r'persons/add/?$', views.PersonCreate.as_view(), name='person_add'),
+    url(r'persons/import/?$', views.PersonImport.as_view(), name='person_import'),
+
+    url(r'^events/import/?$', views.ConferenceImport.as_view(), name='event_import'),
+
+    url(r'^tasks/add/?$', views.TaskCreate.as_view(), name='task_add'),
+    url(r'^tasks/import/?$', views.TaskImport.as_view(), name='task_import'),
+]
```

Comment:
If I understand right, you suggest we make changes within the `workshops` package?
Wouldn't that involve a lot of conditionals? There is a high probability of making the existing code too complex with a lot of branching.